Makefile

```diff
@@ -44,6 +44,11 @@ else
 ZSTDFLAGS := -6
 endif
 
+# Source files used for make targets, evaluated on use.
+GO_SRC_FILES = $(shell find . -not -path './vendor/*' -type f -name '*.go')
+# All the shell files in the repo, excluding ignored files.
+SHELL_SRC_FILES = $(shell shfmt -f . | grep -v "\.coderv2/")
```

Comment:
Since we don't have shfmt installed in every single CI run now is a good opportunity to change this to a `find` instead to avoid "shfmt: command not found" messages in CI.

Reply:
I like `shfmt -f` because it doesn't rely on file extension, can find scripts without as well. But on the flip side, we don't need that feature and if it simplifies CI then why not 👍🏻.